api/types/microvm.proto

```diff
@@ -142,8 +142,6 @@ message Volume {
   VolumeSource source = 4;
   // PartitionID is the uuid of the boot partition.
   optional string partition_id = 5;
-  // Size is the size to resize this volume to.
```

Comment:
I know it's an optional field, but shouldn't we be deprecating this first?

Reply:
We'd need to add `reserved 6`  and probably `reserved "size_in_mb"` to the definition of Volume.